src/main/java/org/sagebionetworks/bridge/sdk/models/upload/UploadFieldType.java

```diff
@@ -0,0 +1,59 @@
+package org.sagebionetworks.bridge.sdk.models.upload;
+
+/**
+ * Represents field types in upload data. This is used to define upload schemas. For more information about Bridge
+ * upload data and field types, see https://sagebionetworks.jira.com/wiki/display/BRIDGE/Bridge+Upload+Data+Format
+ *
+ * @see org.sagebionetworks.bridge.sdk.models.upload.UploadSchema
+ */
+public enum UploadFieldType {
+    /**
+     * Health Data Attachment as a non-JSON blob. The value of this field is a foreign key into the Health Data
+     * Attachments table. Data stored in this format will not be subject to additional post-processing.
+     */
+    ATTACHMENT_BLOB,
+
+    /**
+     * Health Data Attachment as a CSV file. The value of this field is a foreign key into the Health Data Attachments
+     * table. Data stored in this format will be used to construct de-normalized tables during data export.
+     */
+    ATTACHMENT_CSV,
+
+    /**
+     * Health Data Attachment as a JSON blob. The value of this field is a foreign key into the Health Data Attachments
+     * table. Data stored in this format will not be subject to additional post-processing, but is tagged as JSON data
+     * in the schema for convenience.
+     */
+    ATTACHMENT_JSON_BLOB,
+
+    /**
+     * Health Data Attachment as a JSON blob of a specific "table" format. The value of this field is a foreign key
+     * into the Health Data Attachments table. Data stored in this format will be used to construct de-normalizeds
+     * tables during data export.
+     */
+    ATTACHMENT_JSON_TABLE,
+
+    /** A boolean, expected values match Boolean.parse() */
+    BOOLEAN,
+
+    /** A calendar date in YYYY-MM-DD format */
+    CALENDAR_DATE,
```

Comment:
This looks like overlapping with TIMESTAMP.  Is this here for backward compatibility?